README.md

```diff
@@ -5,30 +5,90 @@ Used to automatically upload ProGuard mapping files to [Bugfender](https://bugfe
 
 ## Usage
 
-*Note:* The plugin requires `com.android.tools.build:gradle` in version at least `4.1` to work properly.
```

Comment:
Why removed?

Reply:
This is replaced by the plugin id("com.android.application")

Reply:
I see that they've made some changes indeed. Don't know exactly how this stuff works, so 🤷.